src/common/transactions/transaction-utils.ts

```diff
@@ -15,6 +15,25 @@ import {
   ContractDeployOptions,
   TokenTransferOptions,
 } from '@common/transactions/transactions';
+import {
+  AddressTransactionWithTransfers,
+  CoinbaseTransaction,
+  MempoolTransaction,
+  Transaction,
+  TransactionEventFungibleAsset,
+} from '@stacks/stacks-blockchain-api-types';
+import { displayDate, isoDateToLocalDateSafe, todaysIsoDate } from '@common/date-utils';
+import { getContractName, truncateMiddle } from '@stacks/ui-utils';
+import { stacksValue } from '@common/stacks-utils';
+import { BigNumber } from 'bignumber.js';
+
+type Tx = MempoolTransaction | Transaction;
+
+export type StxTransfer = {
+  amount: string;
+  sender?: string;
+  recipient?: string;
+};
```

Comment:
```suggestion
export interface StxTransfer {
  amount: string;
  sender?: string;
  recipient?: string;
};
```

Generally we stick to `interface` unless we're composing higher level types